src/app/_variables.scss

```diff
@@ -147,3 +141,51 @@ produces
 @function dim-rgb-values-to-rgba($values, $alpha: 1) {
   @return #{'rgba(' + $values + ', ' + $alpha + ')'};
 }
+
+// Theme variables
+:root {
+  // Font
+  --theme-font-primary: 'Open Sans';
+
+  // Base colours
+  --theme-accent-primary: #e8a534;
+  --theme-accent-secondary: #68a0b7;
+
+  --theme-fill-primary: #000; // Base popups, drawers, dropdowns
+  --theme-fill-secondary: #161616; // Layered dropdowns inc search, item actions
+
+  // Fills to tint panels and sub-frames e.g. masterwork/catalyst progress
+  --theme-fill-lighten: rgba(255, 255, 255, 0.15);
+  --theme-fill-darken: rgba(0, 0, 0, 0.35);
```

Comment:
Will have a think. 

I'm considering something like `--theme-panel` and `--theme-panel-nested` as it's a bit more descriptive. 

The flipside is that this also implies structure and I initially envisaged these as more utility styles (rather than semantic styles) since they're used in a few different places that are structured differently

Reply:
Yeah it's tough because we don't really have a solid concept of layering yet. I'd probably start with having styles for (background, sheet, tooltip, item-popup) and then maybe make the layering structure more generic later if we can figure that out.

Reply:
Interesting, I don't think I'd realized that the catalyst progress had a different background (or a title, which really shouldn't be orange...)

Reply:
The heading is easy enough to change if you want? In white:

![image](https://github.com/DestinyItemManager/DIM/assets/1396158/513b7fac-6194-4270-b200-b5b66157e096) ![image](https://github.com/DestinyItemManager/DIM/assets/1396158/57fc62d0-3e03-4f57-a563-b19a20607074)

Or could strip the heading entirely? Although this isn't as clear when there is a kill-tracker… 

![image](https://github.com/DestinyItemManager/DIM/assets/1396158/b08ce812-4fc8-4964-b263-e0cbe5303022) ![image](https://github.com/DestinyItemManager/DIM/assets/1396158/be104fa5-4616-41b0-b442-7b5525ce795c) 
![image](https://github.com/DestinyItemManager/DIM/assets/1396158/53251865-9e52-4c15-98ba-9833f93e49dc)